core/client/app/services/clock.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    run
+} = Ember;
+
+const ONE_SECOND = 1000;
+
+// Creates a clock service to run intervals.
+
+export default Service.extend({
+    second: null,
+    minute: null,
+    hour:   null,
+
+    init() {
+        this.tick();
+    },
+
+    tick() {
+        let now = moment.utc();
+        this.setProperties({
+            second: now.seconds(),
+            minute: now.minutes(),
+            hour:   now.hours()
+        });
+
+        let self = this;
```

Comment:
👍 fine with me. If we need to do this more and more then it may be worthwhile abstracting it a bit to provide consistency and a way to keep some part of the behaviour in place for component integration tests.

ember-concurrency may also help here although I'm not sure how easy/difficult it is to stub generators in tests